app/services/section/preview_service.rb

```diff
@@ -7,6 +7,14 @@ def initialize(user:, manual_id:, section_uuid:, attributes:)
   end
 
   def call
+    manual = Manual.find(manual_id, user)
+    section = if section_uuid
+                manual.sections.find { |sec|
+                  sec.uuid == section_uuid
+                }
+              else
+                manual.build_section(attributes)
+              end
```

Comment:
I'm going to leave this as it is for now. I'm hoping to introduce the `Manual#find_section` method soon at which point this'll probably/hopefully change.